src/modules/QtQml/Timer.js

```diff
@@ -27,14 +27,21 @@ registerQmlType({
         }
     }
 
-    this.start = function() {
-        if (!this.running) {
-            this.running = true;
-            prevTrigger = (new Date).getTime();
+    /* This ensures that if the user toggles the "running" property manually,
+     * the timer will trigger. */
+    this.runningChanged.connect(function() {
+        if (self.running) {
+            prevTrigger = new Date().getTime();
             if (this.triggeredOnStart) {
                 trigger();
             }
         }
+    })
+
+    this.start = function() {
+        if (!this.running) {
```

Comment:
I'm going to go with no... as `runningChanged` won't get fired, so it'll be a no-op.  It was just me changing as little as possible.  There's also a similar no-op check in `this.stop`.

These are both hold-overs from the very first commit in the repo, so as long as you don't want to keep them for sentimental reasons, they can go.